getting_started/step_by_step/exporting.rst

```diff
@@ -115,9 +115,9 @@ changed:
             $AnimatedSprite.flip_v = velocity.y > 0
 
     func _on_Player_body_entered( body ):
-        $Collision.disabled = true
         hide()
         emit_signal("hit")
+        $CollisionShape2D.call_deferred("set_disabled", true)
```

Comment:
It would be better to use `set_deferred("disabled", true)`. But that can be for another PR as I guess the other tutorial needs to be changed too.